bin/src/help.md

```diff
@@ -1,54 +1,59 @@
--i, --input                 Input (alternative to <entry file>)
--o, --output.file <output>  Output (if absent, prints to stdout)
--f, --output.format [es]    Type of output (amd, cjs, es, iife, umd)
--e, --external              Comma-separate list of module IDs to exclude
--g, --globals               Comma-separate list of `module ID:Global` pairs
-                              Any module IDs defined here are added to external
--n, --name                  Name for UMD export
--m, --sourcemap             Generate sourcemap (`-m inline` for inline map)
--l, --legacy                Support IE8
---amd.id                    ID for AMD module (default is anonymous)
---amd.define                Function to use in place of `define`
---no-strict                 Don't emit a `"use strict";` in the generated modules.
---no-indent                 Don't indent result
---environment <values>      Settings passed to config file (see example)
---no-conflict               Generate a noConflict method for UMD globals
---no-treeshake              Disable tree-shaking
---silent                    Don't print warnings
---intro                     Content to insert at top of bundle (inside wrapper)
---outro                     Content to insert at end of bundle (inside wrapper)
---banner                    Content to insert at top of bundle (outside wrapper)
---footer                    Content to insert at end of bundle (outside wrapper)
---interop                   Include interop block (true by default)
+-v, --version Show version number
+-h, --help Show this help message
+-c, --config Use this config file (if argument is used but value
+is unspecified, defaults to rollup.config.js)
+-w, --watch Watch files in bundle and rebuild on changes
+-i, --input Input (alternative to <entry file>)
+-o, --output.file <output> Output (if absent, prints to stdout)
+-f, --output.format [es] Type of output (amd, cjs, es, iife, umd)
+-e, --external Comma-separate list of module IDs to exclude
+-g, --globals Comma-separate list of `module ID:Global` pairs
+Any module IDs defined here are added to external
+-n, --name Name for UMD export
+-m, --sourcemap Generate sourcemap (`-m inline` for inline map)
+-l, --legacy Support IE8
+--amd.id ID for AMD module (default is anonymous)
+--amd.define Function to use in place of `define`
+--no-strict Don't emit a `"use strict";` in the generated modules.
+--no-indent Don't indent result
+--environment <values> Settings passed to config file (see example)
+--no-conflict Generate a noConflict method for UMD globals
+--no-treeshake Disable tree-shaking
+--silent Don't print warnings
+--intro Content to insert at top of bundle (inside wrapper)
+--outro Content to insert at end of bundle (inside wrapper)
+--banner Content to insert at top of bundle (outside wrapper)
+--footer Content to insert at end of bundle (outside wrapper)
+--interop Include interop block (true by default)
```

Comment:
It looks like this formatting is being broken because you haven’t put this inside a code block. Is this intended to be rendered as Markdown?

Reply:
Per Lukas & Kelly's suggestions, I am waiting for #1878 before I continue. Once I do, I will set it up to only run on `*.ts` files, so this `md` formatting will be a non-issue

Reply:
👍